### DISCUSSION TITLE
Lost channel backup and peers force closed channel

### DISCUSSION THREAD

**Discussion by vicalrez** [lightningnetwork/lnd#8351] [General]
Hi, I lost my channel backup and reinstall my node with my 24 seed. I connected my peers abd let them know about the status of my node, they force closed the channels but, one monts later I saw the closed transtation already happened in the menpool but, I still my funds back on chain. There is something I can do?

**Comment by guggero** [comment]
You can use [`chantools sweepremoteclosed`](https://github.com/lightninglabs/chantools/blob/master/doc/chantools_sweepremoteclosed.md). Or, depending on the type of the channels, you might need to pull the anchors first using `chantools pullanchor`. Need more info to say more though.

**Reply by vicalrez** [reply]
Thank you. I opened those channel in my umbrel node. Those are public channels.

**Reply by vicalrez** [reply]
I run this node in a computer with linux (ubuntu).

**Reply by guggero** [reply]
Can you post the channel points please? Otherwise we can't check what the state of the channels are on chain.

**Reply by vicalrez** [reply]
![20240113_102853](https://github.com/lightningnetwork/lnd/assets/125909165/ede1bfc8-3087-41b9-be62-555371bc33f5)

**Reply by vicalrez** [reply]
![20240113_103010](https://github.com/lightningnetwork/lnd/assets/125909165/a3c2a5d4-4791-4ee2-aa0c-86aa0df88d02)

**Reply by guggero** [reply]
Can you copy paste the transaction IDs please? I'm not going to type those from a screenshot.

**Reply by vicalrez** [reply]
Hi, thanks for your support. Transaccion ID 3a48262dd8c8feff805e3c9b0ab329a60527c6f578fb256811624c6f0c542681

**Reply by guggero** [reply]
You should be able to use the `chantools sweepremoteclosed` command mentioned above. Just download `chantools` from here: https://github.com/lightninglabs/chantools/releases/tag/v0.12.1 (make sure to pick the correct version for your operating system and hardware). You can run it on whatever computer you want, it doesn't need to run on Umbrel. The tool will just ask you for your seed and then do its work.

You'd want to run this command:
```
chantools sweepremoteclosed --feerate 75 --publish --sweepaddr <insert_address_from_your_wallet_here_where_funds_should_go_to>
```

**Reply by vicalrez** [reply]
It worked. Thank you so much.
